scripts/mk_all.sh

```diff
@@ -41,4 +41,5 @@ open nat -- need to do something before running a command
 
 #lint_mathlib- only unused_arguments dup_namespace doc_blame ge_or_gt def_lemma instance_priority
   impossible_instance incorrect_type_class_argument dangerous_instance inhabited_nonempty simp_nf
+  simp_var_head
```

Comment:
It doesn't actually matter for this PR or `simp_nf` (since you've fixed all the failures), but this list should be kept up to date with https://github.com/leanprover-community/mathlib/blob/master/scripts/mk_nolint.lean#L26 . Better: we should refactor so the list isn't duplicated. Alternatively we could remove the passing linters from there.

Reply:
Ok, I missed that one.  The `mk_nolint.lean` script should probably be refactored after #2041 anyhow.

Reply:
I've put the refactoring in #2041.  The list is automatically generated in `lint.lean` for both the CI script as well as `mk_nolint.lean`.